modules/core/src/test/scala/org/scalasteward/core/application/CliTest.scala

```diff
@@ -151,27 +155,88 @@ class CliTest extends FunSuite {
     assert(clue(obtained).startsWith("Usage"))
   }
 
-  test("parseArgs: non-default GitLab arguments") {
+  test("parseArgs: non-default GitLab arguments and required reviewers") {
     val params = minimumRequiredParams ++ List(
       List("--gitlab-merge-when-pipeline-succeeds"),
+      List("--gitlab-remove-source-branch"),
```

Comment:
Is this a drive-by? Or a bad merge with main?

Reply:
I think there were some missing test cases (config flags in this case) so I think I just did a bit of improvement here if I remember correctly.